src/components/billing/payment-modal/index.tsx

```
@@ -90,20 +103,63 @@ export default function PaymentModal({
+        {appliedPromotion ? (
+          <VStack fullWidth className={styles.appliedPromotionContainer}>
+            <HStack justify={FlexJustify.Between} align={FlexAlign.Center} fullWidth>
+              <HStack gap={8} align={FlexAlign.Center} >
+                <TicketPercent size={19} />
+                <Typo>
+                  {appliedPromotion.code}
+                </Typo>
+              </HStack>
+              <VStack align={FlexAlign.End}>
+                <HStack gap={6} align={FlexAlign.Center}>
+                <Typo className={styles.promotionDiscount}>{appliedPromotion.discount.toLocaleString()}원 할인</Typo>
```

ESLint

[error] Replace `<Typo·className={styles.promotionDiscount}>{appliedPromotion.discount.toLocaleString()}원·할인` with `··<Typo·className={styles.promotionDiscount}>⏎····················{appliedPromotion.discount.toLocaleString()}원·할인⏎··················`

(prettier/prettier)